include/bitcoin/bitcoin/utility/random.hpp

```diff
@@ -50,10 +57,52 @@ BC_API void pseudo_random_fill(data_chunk& chunk);
  * @param[in]  ratio    The determinant of the minimum duration as the inverse
  *                      portion of the maximum duration.
  * @return              The randomized duration.
+ *
+ * DEPRECATED in v3
  */
 BC_API asio::duration pseudo_randomize(const asio::duration& maximum,
     uint8_t ratio=2);
 
+
+class pseudo_random
+{
+  public:
+    /**
+     * Returns a thread-safe static random generator instance.
+     * @return  The generator.
+     */
+    static std::mt19937& get_twister();
```

Comment:
This should be private.

Reply:
get_twister should be private?  How will it expose the generator?

Reply:
Ah, nevermind, I get it now :+1: